tests/components/edl21/test_config_flow.py

```diff
@@ -0,0 +1,105 @@
+"""Test EDL21 config flow."""
+
+from homeassistant import data_entry_flow
+from homeassistant.components.edl21.const import (
+    CONF_SERIAL_PORT,
+    DEFAULT_DEVICE_NAME,
+    DOMAIN,
+)
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_NAME
+
+from tests.common import MockConfigEntry
+
+VALID_CONFIG = {CONF_NAME: "My Smart Meter", CONF_SERIAL_PORT: "/dev/ttyUSB1"}
+
+
+async def test_show_form(hass) -> None:
```

Comment:
Please take a look at https://github.com/home-assistant/core/pull/88661 and ensure async_setup_entry is always mocked in config flow tests

Reply:
Changed as suggested.